rmf_traffic/test/unit/agv/test_Planner.cpp

```diff
@@ -2855,277 +2856,628 @@ SCENARIO("Test computing Starts from coordinates")
       std::size_t waypoint_index = start_set[i].waypoint();
       CHECK(waypoint_indices.insert(waypoint_index).second);
       CHECK((waypoint_index == 1 || waypoint_index == 2));
-      
       CHECK(start_set[i].location());
+      CHECK(start_set[i].lane());
+      std::size_t& lane_index = start_set[i].lane().value();
```

Comment:
Nitpick: This should be either `std::size_t` or `const std::size_t&` if you don't intend to modify the value.